packages/react-components/react-progress/src/utils/clampValue.ts

```diff
@@ -0,0 +1,15 @@
+export const clampValue = (value?: number, max?: number): number | undefined => {
+  const internalValue = value !== undefined && value < 0 ? 0 : value !== undefined && max && value > max ? max : value;
```

Comment:
You can guarantee that `max` is not `undefined` at this point

```suggestion
export const clampValue = (value?: number, max: number): number | undefined => {
  const internalValue = value !== undefined && value < 0 ? 0 : value !== undefined && value > max ? max : value;
```